scripts/release.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+
+# release.sh will hopefully allow us to publish all of the necessary crates in
+# this repo in the right order. It is assumed that only one person will be
+# releasing all crates at the same time.
+
+set -e
```

Comment:
you should turn on `set -o pipefail` too.

Reply:
actually `set -u` too.

[ref](https://gist.github.com/mohanpedala/1e2ff5661761d3abd0385e8223e16425)

Reply:
[fcdbd93](https://github.com/cosmos/ibc-rs/pull/984/commits/fcdbd93cfd6c43fd2a5ed86a1477a49f4ad27674)